view.js

```diff
@@ -265,10 +272,22 @@ const result = dv
   .flatMap(state.showTree ? tree : (l) => (showParent(l) ? [l] : l.children))
   .filter((l) => !state.onlyWithTasks || someOfMeAndMyChildren(l, showTask))
   .groupBy((l) => (state.groupBySection ? l.link : l.link.toFile()))
-  .sort((g) => (state.sortByPath ? g : g.key.fileName()), "desc");
+  .sort((g) => (state.sortByPath ? g.key.fileName() : g), "desc")
```

Comment:
?

Reply:
Maybe you thought this was a mistake, but it wasn't. If you don't call fileName, then it will sort by file path, instead of just the file name

Reply:
So, this is interesting - I'm happy to revert this change but the way it was originally it did not sort my daily feed in desc chronological order, `2027` would be the second entry, `2023` first, etc...

Why do you think it should be as before?

Reply:
Because it would be wrong this way

Reply:
Path means folder + filename. Filename means only filename